### PR TITLE
DEVDOCS-4088: tax properties

### PR DESCRIPTION
**Pull request by bc-andreadao**
# [DEVDOCS-4088](https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4088)

Merge when ready

This PR is related to [PR 814](https://github.com/bigcommerce/api-specs/pull/814) (the corresponding API spec for this API doc) and [PR 108](https://github.com/bigcommerce-labs/next-dev-center-prototype/pull/108) (redirects)

### CODE REVIEW

File: docs/api-docs/tax/tax-properties.md [hunk 1]
```diff
@@ -0,0 +1,376 @@
...
+No content
+```
+
+<!-- type: tab-end -->
+
+
+
+## Product tax properties 
+
+Tax properties can be associated with products, creating a product tax property. To do so,
```
Comment (traci707) [inline docs/api-docs/tax/tax-properties.md:214]:
```suggestion
Tax properties can be associated with products, creating a product tax property. To do so, specify the product using its product ID. The `product_id` field has the same value as the `id` field from the [Get all products](/api-reference/store-management/catalog/products/getproducts) endpoint. Note that you can add multiple tax properties for a single product.

```

File: docs/api-docs/tax/tax-properties.md [hunk 2]
```diff
@@ -0,0 +1,376 @@
...
+specify the product using its product ID. The `product_id` field has the same value as the `id` field from the [Get all products](/api-reference/store-management/catalog/products/getproducts) endpoint. Note that you can add multiple tax properties for a single product.
```
Comment (traci707) [inline docs/api-docs/tax/tax-properties.md:215]: Remove and add to the line above

File: docs/api-docs/tax/tax-properties.md [hunk 3]
```diff
@@ -0,0 +1,376 @@
...
+
+
+<!-- theme:info -->
+> #### Note
+> Product tax properties are sent when BigCommerce makes a request to a provider for taxes estimations, tax quote commits, and tax quote adjustments. For more info, see the [Tax Provider API](/api-reference/providers/tax-provider-api).
+
+
+### Get product tax properties 
+
+To get the tax properties that are associated with a product, you must specify the `product_id` for the products in the query.
```
Comment (traci707) [inline docs/api-docs/tax/tax-properties.md:294]:
```suggestion
To get the tax properties associated with a product, you must specify the `product_id` for the products in the query. 
```

**Review by traci707 — COMMENTED** [review]
I made a few editorial suggestions.

File: docs/api-docs/tax/tax-properties.md (hunk 2 above)
Comment (bc-andreadao) [reply docs/api-docs/tax/tax-properties.md]: Done

**Comment by bc-andreadao** [comment]
ping @theromulans for possible review

**Comment by bc-andreadao** [comment]
ping @bc-ani here. This is the doc for the Tax Properties and Product Tax Properties endpoint ([CSS-711](https://bigcommercecloud.atlassian.net/browse/CSS-711)) coming out.

**Comment by theromulans** [comment]
Looks great to me, thanks @traci707 and @bc-andreadao 👏

**Review by theromulans — DISMISSED** [review]

File: docs/api-docs/tax/tax-properties.md [hunk 4]
```diff
@@ -0,0 +1,375 @@
...
+
+Tax properties are inputs that merchants can provide into a tax provider's tax calculation to receive more accurate tax calculations from third-party tax providers. These properties are specific to a tax provider and allow merchants and shoppers to have tax-related fields factored into their tax rates. 
+
+The Tax Property V3 API will allow you to perform the following functions:
+- Add tax property fields that are specific to a tax provider (have **variable** inputs for tax calculations)      
+- Associate tax properties to **products**. Unlike tax codes, you can add **multiple** tax properties to a single product
+
+<!-- theme:info -->
+> #### Note
+> Currently, you can associate tax properties to products at the **product level**.
```
Comment (theromulans) [inline docs/api-docs/tax/tax-properties.md:11]: I assume we use language such as "products at the variant level" elsewhere? 👍

File: docs/api-docs/tax/tax-properties.md [hunk 5]
```diff
@@ -0,0 +1,375 @@
...
+
+<!-- theme:info -->
+> #### Note
+> Currently, you can associate tax properties to products at the **product level**. 
+
+For more info, see the [Tax Property V3 API](/api-reference/store-management/tax-properties) reference.
+
+## Tax properties
+
+You must first add tax properties from a tax provider to the store. The tax provider must provide the `code` for a tax property. Both `code` and `display_name` are required.
```
Comment (theromulans) [inline docs/api-docs/tax/tax-properties.md:17]: > You must first add tax properties from a tax provider to the store.

Perhaps this phrasing could use some adjustment.

> You must first add tax properties, as specified by your tax provider, to the store.

🍻 Something like this perhaps?

Comment (bc-andreadao) [reply docs/api-docs/tax/tax-properties.md]: Thanks, that's better wording!

File: docs/api-docs/tax/tax-properties.md (hunk 4 above)
Comment (bc-andreadao) [reply docs/api-docs/tax/tax-properties.md]: Changed the wording to base products to match KB docs

**Comment by bc-andreadao** [comment]
Got a response back about keeping the `id:in` query parameter for GET tax properties. Slack conversation [here ](https://bigcommerce.slack.com/archives/C97B8DEGK/p1659366725512469)

**Review by slsriehl — APPROVED** [review]
great work.

**Pull request merged**